components/AddressButtons/AddressButtons.tsx

```diff
@@ -12,39 +14,56 @@ const AddressButtons: React.FC<Props> = ({
   address,
   showInspectButton = true,
 }) => {
+  const { network } = useContext(NetworkContext);
+
+  const networkType = network.name === 'MAINNET' ? 'mainnet' : 'testnet';
+
+  const openProfileExplorer = () => {
+    window.open(
+      `https://wallet.universalprofile.cloud/${address}?network=${networkType}`,
+      '_blank',
+    );
+  };
+
+  const openAssetExplorer = () => {
+    window.open(
+      `https://wallet.universalprofile.cloud/asset/${address}?network=${networkType}`,
+      '_blank',
+    );
+  };
```

Comment:
Let's keep it simple and use a basic "a" instead of a button with `window.open`

Im not even sure you need these functions here.